packages/blocks/src/_common/components/button.ts

```diff
@@ -128,8 +128,8 @@ export class IconButton extends LitElement {
   @property({ attribute: true, type: Boolean })
   accessor active: boolean = false;
 
-  @property({ attribute: true, type: Boolean })
-  accessor hover: boolean | undefined = undefined;
+  @property({ attribute: true, type: String })
+  accessor hover: 'true' | 'false' | undefined = undefined;
```

Comment:
Or adds fields:
```
reflect: true,
converter: {
  fromAttribute,
  toAttriubute,
}
```

Reply:
Goood

Reply:
Many parts of the codebase use hover=... instead of ?hover as a workaround. We should change that as well. cc @akumatus